cypress/utils/dashboards/commands.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+Cypress.Commands.add('selectFromDataSourceSelector', (dataSourceTitle, dataSourceId) => {
```

Comment:
seeing this as a datasource command as well im not positive what goes into `utils/commands` and what goes into `query_enhancements/commands` which all have a data source command

Reply:
we don't use this one. it is for easy migration.